File: 169-majority-element/majority-element.php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer
     */
    function majorityElement($nums) {
        $map = array();
        $count = array_count_values($nums);
        print_r($count);
        $max_ = max(array_values($count));
        echo($max_);
        foreach($count as $keys => $values){
            if($values == $max_){
                return $keys;
            }
        }
        print_r(array_keys($count));
        
    }
}

File: 169-majority-element/majority-element.py
from collections import Counter
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        counts = Counter(nums)
        value_ = 0
        iter_ = 0
        for value , count in counts.items():
            if  count >= iter_:
                value_ = value
                iter_ = count
        return value_

        